source/Grid/Grid.js

```diff
@@ -63,6 +64,12 @@ export default class Grid extends Component {
     noContentRenderer: PropTypes.func.isRequired,
 
     /**
+     * Optional callback invoked on mouse-wheel activity.
+     * (event: SyntheticEvent): void
+     */
+    onWheel: PropTypes.func,
```

Comment:
yeah it is a bit odd, but then again I think the grid _should_ listen to more of the html events. 

In any case, I'm happy to remove it if it's a blocker for you. So you want me to remove?